src/main/java/mif/vu/ikeea/Controller/LearningDayController.java

```diff
@@ -33,8 +40,14 @@
     }
 
     @GetMapping(path = "/{managerId}/list")
-    public @ResponseBody List<LearningDayResponse> getLearningDaysList(@PathVariable Long managerId) {
-        List<LearningDay> learningDays = learningDayService.getAllByManagerId(managerId);
+    public List<LearningDayResponse> getLearningDaysListSearch(@PathVariable Long managerId,
+                                                               @RequestParam(required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)Date date,
+                                                               @RequestParam(required = false) Long userId, @RequestParam(required = false) Long topicId){
+
+        List<LearningDay> learningDays = learningDayRepository.findAll(Specification.where(LearningDaySpecification.withManager(managerId))
+                .and(Specification.where(LearningDaySpecification.withDate(date)))
```

Comment:
o cia suveiks taip, jei date nepaduos ir cia .and vis tiek dedi? Gal reiktu tikrinti ar ne null ir tada tik .and deti?

Reply:
skipina jei null springas, kiek radau info, ir man veike kelis param padavus